example_dags/ray_taskflow_example_dynamic_config.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+from pathlib import Path
+
+from airflow.decorators import dag, task
+
+from ray_provider.decorators.ray import ray
+
+
+def generate_config():
+    CONN_ID = "ray_conn"
+    RAY_SPEC = Path(__file__).parent / "scripts/ray.yaml"
+    FOLDER_PATH = Path(__file__).parent / "ray_scripts"
+    return {
+        "conn_id": CONN_ID,
+        "runtime_env": {"working_dir": str(FOLDER_PATH), "pip": ["numpy"]},
+        "num_cpus": 1,
+        "num_gpus": 0,
+        "memory": 0,
+        "poll_interval": 5,
+        "ray_cluster_yaml": str(RAY_SPEC),
+        "xcom_task_key": "dashboard",
+    }
```

Comment:
this was just an earlier example .. i may use this approach to make it easy for team members to select from a few preformed specs

i.e. 

config = generate_config(type="gpu_task")
config = generate_config(type="memory_task")
...

this is a very simple example we might go further and spec out cpu/memory requirements, image etc to use for ray workers

Reply:
made an update. please recheck